components/provisioning/warewulf-provision/SOURCES/warewulf-provision.pxe_file_modes.patch

```diff
@@ -36,35 +45,64 @@
              print COOKIE $self->checksum();
              close COOKIE;
              &nprint("Bootstrap image '$bootstrap_name' is ready\n");
---- a/lib/Warewulf/Provision/Pxe.pm	2019-06-17 14:01:40.000000000 -0700
-+++ b/lib/Warewulf/Provision/Pxe.pm	2019-06-25 14:42:31.414866655 -0700
-@@ -94,8 +94,11 @@
+diff --git a/lib/Warewulf/Provision/Pxe.pm b/lib/Warewulf/Provision/Pxe.pm
+index c9f6a7b..c843206 100644
+--- a/lib/Warewulf/Provision/Pxe.pm
++++ b/lib/Warewulf/Provision/Pxe.pm
+@@ -19,6 +19,7 @@ use Warewulf::DataStore;
+ use Warewulf::Provision::Tftp;
+ use File::Basename;
+ use File::Path;
++use File::Path qw(make_path);
```

Comment:
just add `qw(make_path)` to the `File::Path` above?

Reply:
Added your changes. Please forgive my ignorance of Perlisms.